tutorials/getting-started.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: base
+title: Getting Started guide
+---
+
+### Overview
+
+*The Clocker blueprints run in Apache Brooklyn. These instructions assume some familiarity with Apache Brooklyn. Appropriate links are included for those new to Brooklyn. For background information, see [The Theory Behind Brooklyn](http://brooklyn.apache.org/learnmore/theory.html) and the [Brooklyn Getting Started guide](http://brooklyn.apache.org/v/latest/start/).*
+
+To use the Clocker blueprints you have two options:
+
+* use them pre-packaged in Cloudsoft AMP (which is built on Brooklyn with additional enterprise features such as a rich UX, blueprint QA, and commercial support). This gives the simplest and fastest user-experience.
+* use Apache Brooklyn, manually adding the Clocker blueprints.
+
+{::options parse_block_html="true" /}
+
+<ul class="nav nav-tabs">
+    <li class="active amp-tab"><a data-target="#amp, .amp-tab" data-toggle="tab" href="#">AMP</a></li>
+    <li class="brooklyn-tab"><a data-target="#brooklyn, .brooklyn-tab" data-toggle="tab" href="#">Brooklyn</a></li>
+</ul>
+
+<div class="tab-content">
+<div id="amp" class="tab-pane fade in active">
+
+1. [Download AMP](http://www.cloudsoft.io/amp-container-service-early-access)
+2. Install AMP. For more detailed instructions, see the [docs](http://docs.cloudsoft.io/tutorials/tutorial-get-amp-running.html#install-cloudsoft-amp).
+3. Open the AMP UI in your favourite web browser (as per the detailed instructions within the link above) and [setup a location](http://docs.cloudsoft.io/tutorials/tutorial-get-amp-running.html#add-a-location) for where you want to deploy to (e.g. your preferred cloud, or a list of IPs for pre-existing machines). We came up with location's templates to add to your catalog, that you can use out of the box for [AWS](tutorials/locations/aws-example-location.bom), [SoftLayer]((tutorials/locations/sl-example-location.bom)), [Azure](tutorials/locations/azure-example-location.bom), [GCE](tutorials/locations/gce-example-location.bom) and [Blue Box](tutorials/locations/bb-example-location.bom). For more information, head to the [tutorial section](tutorials/swarm-cluster.html#setup-a-cloud-location).
```

Comment:
The location links are broken in https://tbouron.github.io/clocker/tutorials/getting-started.html 
do they need to start with `./`

**softlayer** remove extra `()`